pages/1.13/security/oss/managing-users/local-users/regular-user-accounts/index.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: layout.pug
+navigationTitle: Regular user accounts
+title: Regular user accounts
+excerpt: Managing regular user accounts
+menuWeight: 20
+---
+
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+By default no regular user account exists in DC/OS. The only way to add regular user accounts is via the DC/OS [Identity and Access Management (IAM) API](/1.13/security/oss/iam-api/).
```

Comment:
I don't like the term "regular user accounts" - it makes the external accounts sound irregular. Can we just call them "local user accounts"? @jgehrcke any opinion?

Reply:
Elegance aside, these are two hopefully helpful ways to think technically about user account types:

- A local user account is either a regular user account (human) or a service user account (machine).
- A regular user account is either a local user account (no reference in an external identity provider) or a remote/external user account (reference to external identity provider).

Now we know that the statement "The only way to add regular user accounts is via the DC/OS" is incorrect because a successful single sign-on flow through Auth0 creates an external regular user account in the IAM database.

That is, we totally should call them "local user accounts" here because this is probably what Tim meant to say :-).

The first sentence ("By default no regular user account exists in DC/OS") can be removed I think. It's probably not helping.

Reply:
@jgehrcke Thanks for the clarification. I'm good with service accounts being the "irregular" ones. Although it means that I'll be first against the wall when the machines take over. :robot:

Reply:
Haha, good!

By the way; always happy to get used to other terms as long as we use them consistently :-) It's almost like using a symbol in a math proof and choosing between calling it rho or phi.

Reply:
Renaming to `external user accounts`, `local user accounts` and `service accounts` :rocket: